binding/java/test/ai/picovoice/porcupine/PorcupineTest.java

```diff
@@ -22,17 +22,74 @@
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.Arrays;
 
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class PorcupineTest {
 
+    private static final String ENVIRONMENT_NAME;
     private Porcupine porcupine;
     private String accessKey = System.getProperty("pvTestingAccessKey");
 
+    static {
+        ENVIRONMENT_NAME = Utils.getEnvironmentName();
+    }
+
+    private static String append_language(String s, String language) {
+        if (language == "en")
+            return s;
+        return s + "_" + language;
+    }
+
+    private static String getKeywordPath(String language, String keyword) {
+        return Paths.get(System.getProperty("user.dir"))
+            .resolve("../../resources")
+            .resolve(append_language("keyword_files", language))
+            .resolve(ENVIRONMENT_NAME)
+            .resolve(keyword + "_" + ENVIRONMENT_NAME + ".ppn")
+            .toString();
+    }
+
+    private static String getModelPath(String language) {
+        return Paths.get(System.getProperty("user.dir"))
+            .resolve("../../lib/common")
+            .resolve(append_language("porcupine_params", language)+".pv")
+            .toString();
+    }
+
+    private static String getAudioFilePath(String AuadioFileName) {
+        return Paths.get(System.getProperty("user.dir"))
+            .resolve("../../resources/audio_samples")
+            .resolve(AuadioFileName)
+            .toString();
+    }
+
+    private static String[] getKeywordPaths(String language, String[] keywords) {
```

Comment:
Just to make it more clear:
```suggestion
    private static String[] getTestKeywordPaths(String language, String[] keywords) {
```